error/runtime_spec.go

```diff
@@ -41,6 +41,73 @@ const (
 	// ReadonlyOnWindows represents the error code of readonly setting test on Windows
 	ReadonlyOnWindows
```

Comment:
This could be a more specific `WindowsRootReadonlyFalse`, and the earlier `ReadonlyFilesystem` could be `RootReadonlyImplemented` or some such, to distinguish from other `root.readonly` issues like using a non-boolean value.